Sources/KituraContracts/Contracts.swift

```diff
@@ -22,7 +22,7 @@
  and server side (eg Kitura) of the request (typically a HTTP REST request).
 
  ### Usage Example: ###
- ````
+ ````swift
  router.get("/users") { (id: Int, respondWith: (User?, RequestError?) -> Void) in
      ...
      respondWith(nil, RequestError.notFound)
```

Comment:
/// - Parameter httpCode: a standard HTTP status code
should be 
/// - Parameter httpCode: A standard HTTP status code.